packages/wallets/bitget/src/types.ts

```diff
@@ -0,0 +1,123 @@
+import type { StdSignDoc } from "@cosmjs/amino";
+
+export type BitgetConfig = {
+  covalentApiKey?: string;
+  ethplorerApiKey?: string;
+  blockchairApiKey?: string;
+};
+
+export enum BroadcastMode {
+  /** Return after tx commit */
+  Block = "block",
+  /** Return after CheckTx */
+  Sync = "sync",
+  /** Return right away */
+  Async = "async",
+}
+
+export interface Coin {
+  readonly denom: string;
+  readonly amount: string;
+}
+
+export interface StdFee {
```

Comment:
we should have those types somewhere 🤔

Reply:
this file was redundant copy-paste. removig